### ISSUE TITLE
A11y: An element or widget does not have an accessible name

### ISSUE THREAD

**Issue by huydoan2** [carbon-design-system/carbon#4942]
## What package(s) are you using?

- [ ] `carbon-components`
- [x] `carbon-components-react`

## Detailed description

As described in Bluemix/Bluemix.UnifiedSupport#2745 
 https://github.ibm.com/Bluemix/Bluemix.UnifiedSupport/issues/2745, `MultiSelect` component has some A11y violations.

We are using `carbon-components-react` 7.7.4 through `CloudPal` 1.34.0

For your convenience, here is the screenshot of the issue:
https://media.github.ibm.com/user/47174/files/3746dc00-28af-11ea-897a-9f2d7198d7dc

**Comment by abbeyhrt** [comment]
Hi @huydoan2! Thank you for taking the time to create this issue. Thankfully, we have already documented this problem in #4008. I am going to close this so we don't have any duplicates floating around. Thank you!
